### PR TITLE
updated -added my blog

### PR DESCRIPTION
**Pull request by prank15062002**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Just upload the image that you have mentioned  in the img/blogs folder and you are done :+1:

File: _data/blogs.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@
   title: Interview with a Teacher from Doon International School
   img: yashkumarverma-blog02.jpg
   author_name: Yash Kumar Verma
+  
+  url: https://openblogsource.wordpress.com/2017/12/28/my-journey-open-source
```
Comment (nikit19) [inline _data/blogs.yml]: you need to add "`-`" before url

**Comment by anshumanv** [comment]
Please fill PR template correctly.

**Comment by ParthS007** [comment]
closing thiis as a duplicate of #613

**Pull request closed without merging**